Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.DxfImport.cs

```diff
@@ -239,11 +240,17 @@ public DB.Element CreateDxfImport(List<Base> objects, string fileName, double sc
     var symbol = Doc.GetElement(fam.GetFamilySymbolIds().First()) as DB.FamilySymbol;
     symbol.Activate();
 
+#pragma warning disable CA1031 // Do not catch general exception types
     try
     {
       File.Delete(tempFamilyPath);
```

Comment:
Same comment on `File.Delete` as above: it throws known exceptions so we should catch the ones we can recover from, if any.